Scripts/Macros.cmd.txt

```diff
@@ -0,0 +1,7 @@
+b=%VFS_SCRIPTSDIR%\BuildGVFSForWindows.bat $*
+br=%VFS_SCRIPTSDIR%\BuildGVFSForWindows.bat Release $*
+ut=%VFS_SCRIPTSDIR%\RunUnitTests.bat $*
+utr=%VFS_SCRIPTSDIR%\RunUnitTests.bat Release $*

```

Comment:
Not sure if this one is that useful either

Maybe this isn't everyone's experience, but these are the things I do locally (and I rely on the build agents for everything else)
* Build, 99% of the time it's debug only
* Install what I just built
* Run a specific functional test or test class

Unit tests are my default project in VS, so I don't need a macro for that. Full functional tests take way too long so I rely on the build agents and only run failed ones locally. And there's really no point in running the release versions locally.

Thoughts?

Reply:
That's a fair point. The reason I was offering these as shortcuts was to make very clear how to get started as a dev. I might cut the release variants but I like the idea of here's the basics: build, unit test, functional test. Install probably makes the cut for "how to get productive"

Reply:
Well for a "getting started" scenario, the functional test one is tricky. You have to first build, then install both G4W and GVFS, and _then_ you can run the functional tests. Personally I would tailor this more to our own productivity, and refer new devs to a wiki page.

Reply:
Like it would be awesome to have a macro to run just one functional test, so I don't have to go find the syntax for that every single time :-). e.g. `ft Name.Of.Class.Or.Test`

Reply:
I agree with Saeed's comments completely. We've already got some starter documentation on getting started as a new developer (which could be improved), I'm more interested in optimizing for our productivity.

Reply:
Sounds good, I'll trim this list down.